src/main/kotlin/stickyWallet/StickyWallet.kt

```diff
@@ -0,0 +1,130 @@
+package stickyWallet
```

Comment:
Not a conventional package name but who cares if it works ig

Reply:
I mean you're right, it should've been `com.dumbdogdiner.stickyWallet`, however... the amount of nesting makes me wanna rage soo... If you want it, feel free to PR <3